### PR TITLE
Provide a toggle to turn on display of seek graph

### PR DESCRIPTION
**Pull request by GreenAsJade**
Fixes seek graph taking up unwelcome space on Custom Games page

## Proposed Changes

  - Provide a toggle to turn it on for people who want to see it.
  

Note: some shenanigans are required to keep seek graph rendered, so we don't loose the ref to it.

### CODE REVIEW

File: src/views/Play/CustomGames.tsx [hunk 1]
```diff
@@ -74,6 +74,12 @@ export function CustomGames(): JSX.Element {
     const canvas: HTMLCanvasElement = React.useMemo(() => allocateCanvasOrError(), []);
     const seekgraph = React.useRef<SeekGraph>();
 
+    const [seekGraphVisible, setSeekGraphVisible] = React.useState(false);
```
Comment (anoek) [inline src/views/Play/CustomGames.tsx:77]: Maybe should do a `usePreference` here instead of `useState` so we remember their setting?

Comment (GreenAsJade) [reply src/views/Play/CustomGames.tsx:77]: Doh yes!

**Comment by BHydden** [comment]
Wouldn't default on be less abrasive amidst some other controversial UI changes? 😅

**Comment by GreenAsJade** [comment]
I haven't seen anyone arguing in favour of the seek graph, and it's position seems to be agreed as "making it even harder to find the custom games".

**Comment by BHydden** [comment]
Personally I agree, if it were me I would scrap it. But I seem to remember past forum discussions about that meeting resistance. Lemme see if I can find any.

**Comment by GreenAsJade** [comment]
I think anyone desperate that we keep it should be satisfied that in one click it will remain there for ever more for them

**Pull request merged**